sdks/open-format/src/core/contract.ts

```diff
@@ -421,7 +423,8 @@ export async function deploy({
     nft.symbol,
     nft.url,
     nft.maxSupply,
-    ethers.utils.parseEther(nft.mintingPrice.toString())
+    ethers.utils.parseEther(nft.mintingPrice.toString()),
+    transactionArgs && { transactionArgs }
```

Comment:
In @tinypell3ts original implementation the arguments were spread.

```tsx
  const contract = await openFormatContract.deploy(
    nft.name,
    nft.symbol,
    nft.url,
    nft.maxSupply,
    ethers.utils.parseEther(nft.mintingPrice.toString()),
    { ...transactionArgs }
  );
```

Pretty sure this is now passing the last argument of `deploy` to be an object with the args inside.

Reply:
Fixed 🙌